pkg/warmup/target.go

```diff
@@ -77,6 +78,11 @@ func (t Target) WaitForReadinessProbe() error {
 			} else {
 				request, err := grpc.ToGrpcRequest(t.options.ReadinessGrpcMethod)
 				if err == nil {
+					log.Print("gRPC readiness client connecting...")
```

Comment:
Since the gRPC client handles reconnection by nature we just need to establish this connection once for the readiness client (if a gRPC readiness is used) and once for the gRPC requests.